### DISCUSSION TITLE
Please take part of the next dev surveys for esp32_8ch_motor_shield

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#66] [Announcements]
Hi everybody, 

You have take part of that project as starring it, contributing or posting comment in discussion. 

Now, we need you for many surveys to dev the next steps. 

Please look at  https://github.com/nliaudat/esp32_8ch_motor_shield/discussions and fill the surveys. 

If you have suggestions or want to help. Feel free to open a new discussion. 
###########
Please, do not post in that discussion
############

Regards

@WMP @Karl-opec @rzerres  @MarcinJanowski  @yurii-a @designia @ @fgbs @liselotte111 @erwin314 @floatyboy @apeman76 @GermanDZ @Mosher23 @grericht @william-eu @pimdev @jonathanjacob @rokgerzelj @chriszero @edomat @MrAges @tianhucn @Snypes88 @BartVB @MAButz @DerMo92 @petrows @gcpinto @thanoskas @wunderbaum @herrfrei @shnhrrsn @mkaiser @Surstroemming2000 @slowmow766 @hydromac @MP91 @THorst92 @MarcHagen @Jurian15 @deku-m @0bmxa @zazusy @BernieHD @cregganna @pasarn @rici44 @benv666 @dschoorisse @dragop4 @xdubx @Me-xx @ewertmaster @qba667 @velijv @MikolajPieklo @PatrickEekhout @tango38317 @10bn @scyomantion @gustavstrandberg @Tecnologic @verstrepeng @pietervanstee @AlexanderNaehring @EelcoBaartse @noah80 @LinFor @lal12 @whins @MrG00dkat @d00616 @amateescu @Scarbous @a005 @misery @bazz46 @Gorgonbert @HINATASDK @ridikuel @kralikba @baranator @croghostrider @konwin @drivingtheproject @mo8Zomo0 @malkowitz @rubenmuehlhans @Zwer2k @trtracer @kaskerol @ckohrt

**Comment by mkaiser** [comment]
What about another housing, e.g. Phoenix Contact BC 107,6. Costs around 13€

**Reply by mkaiser** [reply]
Another thought: i am Using some kind of CAN Bus for connecting my home iot devices. 

Could you foresee a CAN transceiver (Most esp32 devices have built-in Controllers) in the PCB as an assembly option? 

E.g. https://www.mouser.de/ProductDetail/Texas-Instruments/SN65HVD230MDREP?qs=7kv20MEVUlh1xXFkHDCt7w%3D%3D

This would allow me to integrate it via https://github.com/mrk-its/esphome-canopen

**Reply by nliaudat** [reply]
Canbus - modbus is not really linked to floor heating systems.
I suggest you one of my other board : https://github.com/nliaudat/esp_canbus 
You can adapt it for your needs as it can handle all of CAN messages with his SN65HVD230  Transceivers

**Reply by mkaiser** [reply]
My whole house is running on custom made PCBs connected via canbus (roller/shutter & lights)

with the esphome-canopen approach (this project makes much progress!) it hopefully will be possible to connect to HomeAssistant via wired CAN, instead of Wifi (more reliable and less power consuming). 

> I suggest you one of my other board : https://github.com/nliaudat/esp_canbus

I created a similar project some time ago for my CAN-based heating-pump :)
https://github.com/mkaiser/ESP32-CAN

One assembly option for a CAN transceiver would be sufficient for me :)

**Reply by nliaudat** [reply]
You need only a connector with 4 gpios (CAN-RX, CAN-TX, 3V3, GND)
I'll add a connector for i2c as asked in https://github.com/nliaudat/esp32_8ch_motor_shield/discussions/67
It could fit your needs
